src/ResourceManagement/Dns/TxtRecordSetsImpl.cs

```diff
@@ -16,24 +16,24 @@ internal partial class TxtRecordSetsImpl :
         DnsRecordSetsBaseImpl<ITxtRecordSet, TxtRecordSetImpl>,
         ITxtRecordSets
     {
-        ///GENMHASH:DE142040F109CD3B229E5070C0CA4D4F:F8E87D142BE7B967C3D37E08C8777506
-        internal TxtRecordSetsImpl(DnsZoneImpl dnsZone) : base(dnsZone, RecordType.TXT)
+        ///GENMHASH:DE142040F109CD3B229E5070C0CA4D4F:3D4AA50D3AD17F26B290CAD1D1D863EA
+        internal TxtRecordSetsImpl(DnsZoneImpl dnsZone) 
+			: base(dnsZone, RecordType.TXT)
         {
         }
 
-        ///GENMHASH:5C58E472AE184041661005E7B2D7EE30:891E91CBEE046376B0D9377B27399767
-        public async override Task<ITxtRecordSet> GetByNameAsync(string name, CancellationToken cancellationToken = default(CancellationToken))
+        ///GENMHASH:B94D04B9D91F75559A6D8E405D4A72FD:F9073E112F17454F48F8E8DCD9BA7F1A
+        protected override IEnumerable<ITxtRecordSet> ListIntern(string recordSetNameSuffix, int? pageSize)
         {
-            RecordSetInner inner = await dnsZone.Manager.Inner.RecordSets.GetAsync(
-                dnsZone.ResourceGroupName,
-                dnsZone.Name,
-                name,
-                recordType,
-                cancellationToken);
-            return new TxtRecordSetImpl(dnsZone, inner);
+            return WrapList(Extensions.Synchronize(() => dnsZone.Manager.Inner.RecordSets.ListByTypeAsync(dnsZone.ResourceGroupName,
+                                                                dnsZone.Name,
+                                                                recordType,
+                                                                top: pageSize,
+                                                                recordsetnamesuffix: recordSetNameSuffix))
+                                                    .AsContinuousCollection(link => Extensions.Synchronize(() => dnsZone.Manager.Inner.RecordSets.ListByTypeNextAsync(link))));
```

Comment:
indentation is in-correct [ref](https://github.com/hovsepm/azure-libraries-for-net/blob/7eec7c75efa4d0157ba158acc43d240708ed1443/src/ResourceManagement/Dns/TxtRecordSetsImpl.cs#L46)